ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitSendDetailsWidget.kt

```diff
@@ -0,0 +1,60 @@
+package org.p2p.uikit.components
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import org.p2p.uikit.databinding.WidgetSendDetailsInputBinding
+import org.p2p.uikit.textwatcher.AmountFractionTextWatcher
+import org.p2p.uikit.utils.inflateViewBinding
+
+class UiKitSendDetailsWidget @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    private val binding: WidgetSendDetailsInputBinding = inflateViewBinding()
+
+    var amountListener: ((input: String) -> Unit)? = null
+    val switchListener: (() -> Unit)? = null
+    val tokenClickListener: (() -> Unit)? = null
+
+    val imageViewTokenIcon
+        get() = binding.imageViewTokenIcon
```

Comment:
The view should make its fields public. 
If we need to load images into it, let's inject GlideManager here and create a method: 

`fun loadImage(imageUrl: String)`

The only thing is to move Glide manager